### PR TITLE
Braden v2.1 Documentation fixes

### PR DESCRIPTION
**Pull request by BradenTeo**
- Fix my previously added sequence diagram
- Change `<<class>>` of other diagrams to be red
- Fix some activation bar issues in 2 diagrams

PS I removed the max message limit for ShowScheduleForADateSDOne.png because it messed with the activation bar

### CODE REVIEW

**Review by yuzhongng — APPROVED** [review]

File: docs/puml/ShowScheduleForADateSDOne.puml [hunk 1]
```diff
@@ -1,23 +1,17 @@
 @startuml
 
-skinparam maxMessageSize 85
-
```
Comment (jltha) [inline docs/puml/ShowScheduleForADateSDOne.puml:4]: I originally had the message size limit to increase user readability. But If you really want to remove it, you may remove that just line. As for the rest of the code in the SD, please undo the changes made. The diagram will autocorrect itself with the correct activation bar when a new ShowCommand is created

**Review by jltha — CHANGES_REQUESTED** [review]

**Review by jltha — APPROVED** [review]

**Review by thedarie — APPROVED** [review]

**Comment by BradenTeo** [comment]
@hubot /deploy
